css/worksection.css

```diff
@@ -98,6 +96,69 @@
   color: #fff;
 }
 
+#modal-overlay {
```

Comment:
- [ ] The popup is not responsive on the mobile view, kindly fix this

![Screenshot_20220513_232325](https://user-images.githubusercontent.com/25227022/168396711-e399e4cd-c68d-455c-8eab-d851eaf84bcf.png)